robot_controll/main.cpp

```diff
@@ -209,10 +209,12 @@ int main(int argc, char** argv)
     //sm.loc->readMap( (char *)"../maps/stromovka.osm" );
     //sm.loc->readMap( (char *)"../maps/lodz.osm" );
     //sm.loc->readMap( (char *) "../maps/homologacie_fei.osm" );
-    sm.loc->readMap( (char *) "../maps/deggendorf.osm" );
+    //sm.loc->readMap( (char *) "../maps/deggendorf.osm" );
+    //sm.loc->readMap( (char *) "../maps/sad_jk.osm" );
     //sm.loc->readMap( (char *)"../maps/botanicka.osm" );
     //sm.loc->readMap( (char *)"../maps/borsky2.osm" );
     //sm.loc->readMap( (char *)"../maps/pisek.osm" );
+    sm.loc->readMap( (char *)"../maps/matfyz.osm" );
```

Comment:
@pavelpetrovic I strongly hope there comes a time when changing this will not require a full recompilation of the whole `.cpp` file.